### ISSUE TITLE
Item details should be more obvious or simply listed when you click through

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#56]
In order to find out more information on particular objects, it seemed that many users could not find the Item Details button. This may need to be made more obvious or the information listed there should be included on the results page when you click through to the item.
